src/createServer.js

```diff
@@ -1,3 +1,72 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const BASE = 'http://localhost:5701';
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const reqUrl = new URL(BASE + req.url, BASE);
```

Comment:
I am not sure that such a link will work correctly, have you tried running your program?
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
It works (the URL object builds its params correctly), though I will change the first argument to match the examples.